gui/gui.go

```diff
@@ -266,27 +266,26 @@ func (ui *GUI) Run(ctx context.Context) {
 		}
 
 		if ui.cfg.UseLEHTTPS {
-			certCache := autocert.DirCache("certs")
 			certMgr := &autocert.Manager{
 				Prompt:     autocert.AcceptTOS,
-				Cache:      certCache,
+				Cache:      autocert.DirCache("certs"),
 				HostPolicy: autocert.HostWhitelist(ui.cfg.Domain),
 			}
 
-			// Ensure port 80 is not already in use.
-			port80 := ":80"
-			listener, err := net.Listen("tcp", port80)
+			// Ensure the gui port is not already in use.
```

Comment:
When we discussed this on Matrix, I thought the idea was to remove the http redirect completely, rather than just changing it from port 80 to the port provided in config. Can we remove it completely? I don't think its really needed, all clients should be expected to use https.

Reply:
I agree after taking another look, clients should be expecting https and if the redirect is needed it can be handled by the pool operator's web server. The redirect shouldn't be default as a result.